scripts/zones/Maze_of_Shakhrami/Zone.lua

```diff
@@ -21,7 +22,13 @@ function onInitialize(zone)
     SetGroundsTome(tomes);
 
     UpdateTreasureSpawnPoint(17588769);
-    
+
+    local whichNM = math.random(0,19);
+    if (whichNM < 10) then
+        SetRespawnTime(Argus, 0, 43200); -- 0-12 hours 
+    else
```

Comment:
I hope I got this right, because according to luautils.cpp:

/************************************************************************
    *                                                                       *
    * Set SpawnType of mob to scripted (128) or normal (0) usind mob id     *
    *                                                                       *
    ************************************************************************/
    int32 DeterMob(lua_State* L)

I set both of them to 128 in the db so I don't have to call DeterMob here